app/popup.html

```diff
@@ -8,7 +8,10 @@
     <link rel="stylesheet" type="text/css" href="./index-rtl.css" title="rtl" disabled>
   </head>
   <body style="width:357px; height:600px;">
-    <div id="app-content"></div>
+    <div id="app-content">
+      <img class="loading-logo" src="./images/logo/metamask-fox.svg" alt="" />
```

Comment:
I think it would liven up the experience if this were the 3D, interactive fox that appears on the login screen, but that can be tweaked later.

Reply:
Actually, besides this, one thing I am noticing is that in the usual case when no error occurs upon initialization, when you open the popup, the loading screen briefly flashes on the screen before showing the home screen. I feel like we could tighten this up a bit. What do you think about wrapping these two divs in another div with a class of `hidden` (not sure if this is defined anywhere, but it would be `display: none`), and adding a `setTimeout` which will wait 1 second or so before removing the `hidden` class? This should remove that flash.

Reply:
@mcmire can we do this in another PR, so we can involve the design team too. The current behaviour is that the text "Loading" briefly flashes....this inherits that behaviour and would be nice to resolve it properly.

Reply:
Ah ok no problem!